cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandlerInternal.java

```diff
@@ -73,8 +72,8 @@ public void getProgramRunRecordMeta(HttpRequest request, HttpResponder responder
                                       @PathParam("program-name") String programName,
```

Comment:
@albertshau I'm thinking if we can remove the version in PathParam? Since it's internal API and the caller is calling from [RemoteProgramRunRecordFetcher](https://github.com/cdapio/cdap/blob/develop/cdap-watchdog/src/main/java/io/cdap/cdap/logging/gateway/handlers/RemoteProgramRunRecordFetcher.java#L61) Another interesting finding is that... the plural [RemoteProgramRunRecordsFetcher](https://github.com/cdapio/cdap/blob/develop/cdap-watchdog/src/main/java/io/cdap/cdap/logging/gateway/handlers/RemoteProgramRunRecordsFetcher.java#L52) is using V3 public API and the singular is using V3 internal... any reasons we have difference?